sudoku/general/modal.js

```diff
@@ -0,0 +1,45 @@
+import { answers, check } from '../easy/script.js';
+
+import { reset } from './stopwatch.js';
+let modal = document.querySelector('.modal');
+let span = document.querySelector('.modal>span');
+let scoreNames = document.querySelectorAll('.name>li');
+let scoreSecs = document.querySelectorAll('.score>li');
+let res = [];
+let name = document.querySelector('#name');
+let modalOk = document.querySelector('.modalBtn');
+const reg = /\d\d\:\d\d\:\d\d/;
```

Comment:
please chose more descriptive name. The name should describe what this regexp is doing